src/server/services/service/resources/transformers/resources-by-environment.js

```diff
@@ -4,36 +4,22 @@ import { sortByName } from '../../../../common/helpers/sort/sort-by-name.js'
 import { renderComponent } from '../../../../common/helpers/nunjucks/render-component.js'
 import { noValue } from '../../../../common/constants/no-value.js'
 
-const keyCellClass = 'app-summary-list__key'
-
-const buildSubscriptionsList = (subscriptions) => {
```

Comment:
@christopherjturner tracing back through the code it looks like its intended. But I wanted to check. Subscriptions on a queue used to be an array of objects. This is now an array of strings. I've updated accordingly but wanted to double check that this is correct?

Previously it was:
```
"subscriptions": [
  {
    "filterEnabled": false,
    "filterPolicy": "",
    "topics": [
      "error_notification.fifo"
    ]
  }
],
```

Now its:

```
"subscriptions": [
  "error_notification.fifo"
],
```